meta/pdir/social-feed-bundle/en.yml

```diff
@@ -1,9 +1,18 @@
 en:
     title: Social Feed
     description: >
-        The Social Feed Extension shows a user feed from the most popular social networks. Currently the extension supports Facebook and Instagram.
+        The Social Feed Extension shows a user feed from the most popular social networks. Currently the extension
+        supports Facebook and Instagram.
```

Comment:
Twitter nicht? In der deutschen Beschreibung ist auch Twitter angeführt.

Reply:
Doch, habe ich übersehen. Danke für den Hinweis.